toggles/webapp/toggles.js

```diff
@@ -21,53 +20,12 @@ module.exports = {
       defaultValue: false,
       description: 'Shows all possible search results from the catalogue API',
     },
-    {
-      id: 'refineFiltersPrototype',
-      title: 'Display the Refine filters prototype',
-      defaultValue: false,
-      description:
-        'Shows filters on the works search pages. Geared towards refinement.',
-    },
     {
```

Comment:
As discussed, we can remove this now, since we're decided on the direction (just need to fix the implementation). Or we can do it in another PR.